extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -38,13 +40,45 @@ export const buildOpenAttachmentElement = (element) =>
     </a>`;
 
 /**
- * Renders the open attachment UI affordance.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenInlineAttachmentElement = (element) =>
+  htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button">
+      <div class="i-amphtml-story-inline-page-attachment-chip">
+        <div class="i-amphtml-story-inline-page-attachment-img"></div>
+        <div class="i-amphtml-story-inline-page-attachment-arrow"></div>
+      </div>
+    </a>`;
+
+/**
+ * Determines which open attachment UI to render.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+export const renderPageAttachmentUI = (win, pageEl, attachmentEl) => {
+  const openImgAttr = attachmentEl.getAttribute('cta-image');
+  const attachmentHref = attachmentEl.getAttribute('href');
+  if (isPageAttachmentUiV2ExperimentOn(win) && !attachmentHref && openImgAttr) {
+    return renderPageAttachmentUiWithImages(win, pageEl, attachmentEl);
```

Comment:
It would be good to test that this codepath runs and correct UI renders.

We could write two tests for this: 
- Renders new UI if the experiment is on and one image is passed
- Renders new UI if the experiment is on and two images is passed

Reply:
done!